xml/System.IO.Compression/ZipArchive.xml

```diff
@@ -299,9 +299,19 @@ An encoding other than UTF-8 is specified for the <paramref name="entryNameEncod
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets the optional archive comment.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The comment encoding is determined by the `entryNameEncoding` parameter of the <xref:System.IO.Compression.ZipArchive.#ctor(System.IO.Stream,System.IO.Compression.ZipArchiveMode,System.Boolean,System.Text.Encoding)> constructor.
```

Comment:
@gewarren this line, and the similar one in ZipArchiveEntry.xml, seem to have a problem, but I'm not sure what it is. Is it that xrefs do not allow `#ctor` to indicate constructor?